packages/kbn-test/src/kbn_client/kbn_client_saved_objects.ts

```diff
@@ -219,6 +219,25 @@ export class KbnClientSavedObjects {
     this.log.success('deleted', deleted, 'objects');
   }
 
+  public async cleanStandardList(space: string = '') {
+    // add types here
+    const types = [
+      'search',
+      'index-pattern',
+      'visualization',
+      'dashboard',
+      'lens',
+      'map',
+      'graph-workspace',
+      'query',
+      'tag',
+      'url',
+      'canvas-workpad',
+    ];
+    const options = { types, space };
+    await this.clean(options);
+  }
```

Comment:
I think we should use an options object here, positional arguments in global APIs like this often cause problems and large refactors when requirements change down the road.

```suggestion
  public async cleanStandardList(options?: { space?: string }) {
    // add types here
    const types = [
      'search',
      'index-pattern',
      'visualization',
      'dashboard',
      'lens',
      'map',
      'graph-workspace',
      'query',
      'tag',
      'url',
      'canvas-workpad',
    ];
    const options = { types, space: options?.space };
    await this.clean(options);
  }
```